### PR TITLE
adding a test ...

### PR DESCRIPTION
**Pull request by gabiudrescu**
...to ensure AbstractContainerAwareStep is implementing ContainerAware interface

### CODE REVIEW

File: src/Sylius/Bundle/FlowBundle/spec/Process/ProcessSpec.php [hunk 1]
```diff
@@ -28,6 +28,11 @@ function it_is_a_process()
         $this->shouldImplement('Sylius\Bundle\FlowBundle\Process\ProcessInterface');
     }
 
+    function it_is_container_aware()
+    {
+        $this->shouldImplement('Symfony\Component\DependencyInjection\ContainerAwareInterface');
```
Comment (tristanbes) [inline src/Sylius/Bundle/FlowBundle/spec/Process/ProcessSpec.php]: isn't that deprecated ? Isn't the new recommended thing is to use the ContainerAwareTrait ?

Comment (pamil) [inline src/Sylius/Bundle/FlowBundle/spec/Process/ProcessSpec.php]: I've just added this to 2.8 upgrade list, only abstract class `ContainerAware` is deprecated, symfony/symfony#17061.

**Comment by pjedrzejewski** [comment]
Could you take a look at the failing build? Also, you can use ``::class``.

**Comment by gabiudrescu** [comment]
yeah, now I realise it failed because src/Sylius/Bundle/FlowBundle/spec/Process/ProcessSpec.php is a spec file for src/Sylius/Bundle/FlowBundle/Process/Process.php not for src/Sylius/Bundle/FlowBundle/Process/Step/AbstractStep.php or src/Sylius/Bundle/FlowBundle/Process/Step/AbstractContainerAwareStep.php

shouldn't there be a StepSpec.php?

**Pull request closed without merging**